src/components/outlier-analysis-table/OutlierAnalysisTable.js

```diff
@@ -118,37 +174,46 @@ OutlierAnalyisTable.generateElementKey = e =>
     }-${e.dataElementId}`
 
 OutlierAnalyisTable.convertElementFromApiResponse = e => ({
```

Comment:
So from what I can gather this method is used in `src/pages/outlier-detection/OutlierDetection.js`. Not something you can do anything about, but seems so confusing that this is defined here as a static method. But yeah, guess it fits with the other patterns used in this app.